bin/testObservability/helper/helper.js

```diff
@@ -55,6 +55,7 @@ const httpsScreenshotsKeepAliveAgent = new https.Agent({
 });
 
 const supportFileCleanup = () => {
+  return;
```

Comment:
why are to return from here at the start ?

Reply:
Removed